FirebaseStorageSwift/Tests/Unit/StorageTests.swift

```diff
@@ -181,12 +194,22 @@ class StorageTests: XCTestCase {
     let options = FirebaseOptions(googleAppID: "0:0000000000000:ios:0000000000000000",
                                   gcmSenderID: "00000000000000000-00000000000-000000000")
     options.projectID = "myProjectID"
-    if bucket != "nil" {
-      options.storageBucket = bucket
-    }
+    options.storageBucket = bucket
     let name = "StorageTests\(bucket)"
     let app = FirebaseApp(instanceWithName: name, options: options)
     StorageTests.appDictionary[bucket] = app
     return app
   }
+
+  private func XCTAssertThrowsObjCException(_ closure: @escaping () -> Void) {
+    XCTAssertThrowsError(try ExceptionCatcher.catchException {
+      closure()
+    })
+  }
+
+  private func XCTAssertNoThrowObjCException(_ closure: @escaping () -> Void) {
+    XCTAssertNoThrow(try ExceptionCatcher.catchException {
+      closure()
+    })
+  }
```

Comment:
Love how these turned out :D 

Small nit picky question: can we just pass in the closure directly like `ExceptionCatcher.catchException(block: closure)` or will that not work?

Reply:
Do you mean 
```
    XCTAssertThrowsError(try ExceptionCatcher.catchException( {
      closure()
    }))
```
Don't the extra parens make it less readable?

Reply:
Just checked and we can use this:
```
XCTAssertThrowsError(try ExceptionCatcher.catchException(closure))
```

The idea is that the argument expected in `catchException` is a closure that takes no arguments and has no return type, which is exactly what the `closure` variable we have is. So instead of using the trailing closure syntax to create a new closure that only calls the function `closure()` that we passed in, we can pass it directly as an argument (like we would a selector in ObjC)